spec/support/vacancy_helpers.rb

```diff
@@ -134,6 +134,12 @@ def verify_vacancy_list_page_details(vacancy)
     expect(page.find('.vacancy')).to have_content(vacancy.publish_on)
     expect(page.find('.vacancy')).to have_content(vacancy.starts_on)
     expect(page.find('.vacancy')).to have_content(vacancy.working_patterns)
+
+    # rubocop:disable Style/GuardClause
+    unless vacancy.expiry_time.nil?
+      expect(page.find('.vacancy')).to have_content(vacancy.expiry_time.strftime('%H:%M%P'))
```

Comment:
This might need to be updated according to the previous comment

Reply:
fixed in b4a7425